README.md

```diff
@@ -291,7 +291,7 @@ You can use your own assets instead but for this tutorial, I'll use Unreal Engin
 Now go to "Game/Async Loading Screen" setting in the Project Settings and open "Startup Loading Screen" section. This Startup Loading Screen only shows once when the player first opens the game so usually, you don't want to show a loading screen here. Instead, you'll want to show a UE4 logo intro movie, your studio logo movie, your sponsor 1 logo movie, your sponsor 2 logo movie, or some weird shit movies, depending on you.
 
 - Uncheck "Auto Complete when Loading Completes" option so we can see the movies play till end
-- Add your movie path without extension in "Movie Paths" array
```

Comment:
This is pretty deceptive, you literally MUST NOT use a path in order for this to work hahaha! You must specify the filename without extension only.